helix-term/src/ui/completion.rs

```diff
@@ -11,8 +11,10 @@ use std::borrow::Cow;
 use helix_core::{Position, Transaction};
 use helix_view::Editor;
 
-use crate::commands;
-use crate::ui::{menu, Markdown, Menu, Popup, PromptEvent};
+use crate::{
+    commands,
+    ui::{menu, Markdown, Menu, Popup, PromptEvent},
+};
```

Comment:
Why not keep this?

Reply:
Me too, again, it's the default behaviour `cargo fmt` overwriting what was there before; I don't know what options were used to autoformat the previous variant, but if someone has them I'm happy to re-run it.

Reply:
We're just using out of the box cargo fmt settings. Are you sure you're not on nightly or have a global fmt config? There's an option to merge imports but it was too eager so it was disabled by default in fmt.

Reply:
Ah, I think you caught me with nightly, that must be it! I'll run it from stable and retry